test_mode: "hover" # circular, square, eight
takeoff_height: 1.0


# Hover mode
hover_duration: 60.0


# Circular
radius: 2.0
